node/clients/pricing.ts

```diff
@@ -0,0 +1,33 @@
+import type { InstanceOptions, IOContext } from '@vtex/api'
+import { ExternalClient } from '@vtex/api'
+
+export class Pricing extends ExternalClient {
+    constructor(context: IOContext, options?: InstanceOptions) {
```

Comment:
Why don't we get the price directly from the product

Reply:
The client is removed.